From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: leguan <longboard.noah@gmail.com>
Date: Wed, 27 Aug 2025 18:06:22 +0200
Subject: [PATCH] setup


diff --git a/build.gradle.kts b/build.gradle.kts
index 8d78a2c5db2bf5896fcf53baf972f289320ad5b8..333252052d11cbedb7a9af72d2573370b428b1e9 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -5,12 +5,12 @@ plugins {
 }
 
 dependencies {
-    compileOnly(project(":api"))
+    compileOnly(project(":staticpaper-asp-api"))
     compileOnly(paperApi())
     implementation(libs.zstd)
 }
 
 publishConfiguration {
-    name = "Advanced Slime Paper Core"
+    name = "StaticPaper ASP Core"
     description = "Core logic for Advanced Slime Paper"
 }
